BUILD

```diff
@@ -226,6 +227,7 @@ cc_library(
     "src/core/json/json_common.h",
     "src/core/json/json_reader.h",
     "src/core/json/json_writer.h",
+    "src/core/proto/load_balancer.pb.h",
```

Comment:
The problem with making the generated files go there is that the nanopb generated pb.{c,h} will clash with the ones generated by the regular C++ proto compiler. One or the other ought to output their generated code to a non-standard (ie, not following the proto package hierarchy) directory.